### ISSUE TITLE
least diverged ortholog

### ISSUE THREAD

**Issue by cmungall** [BgeeDB/homology-ontology#1]
we have been mapping PANTHER LDO to
http://purl.obolibrary.org/obo/HOM_0000020

is this correct?

**Comment by marcrr** [comment]
Interesting case. There is no concept of Least diverged ortholog in HOM at present.

The mapping to HOM0000020 "1 to 1 orthology" is not correct, since the description in PantherDB (http://www.pantherdb.org/genes/) implies that there is more than one ortholog to chose from.

So first, these LDOs map to HOM0000048 "many to many orthology".

Secondly, it can be discussed whether they should be also mapped to HOM0000054 "isoorthology". If we accept the reasoning in PantherDB, this seems to me to be the best mapping.

LSO PantherBD: "PANTHER also infers the “least diverged orthologs” (LDOs), which are the genes in two different organisms that have diverged the least since their common ancestor and are most likely to retain the greatest functional similarities."

HOM0000054: "Orthology that involves functional equivalent genes with retention of the ancestral function." This is the corresponding sentence from Fitch 2000: "It would be appropriate, whenever a pair of paralogous genes are both orthologous to a more distant gene – and they diverge such than one form retains the old function while the other acquires a new function – to label the pair of orthologs retaining the same function as isorthologs (from iso, meaning same)."

I think that we should ask the PantherDB team whether they feel that isoorthology covers their LSO concept, or whether an LSO concept should be added to the HOM.

**Comment by tarcisiotmf** [comment]
Actually, in a parallel work to structure data from multiple orthology databases (e.g. OMA, orthoDB, in paranoid, Hieranoid, etc.) , we have discussed about including the LOD term (i.e. LeastDivergedOrthologyRelation) in the next release of ORTH ontology (https://github.com/qfo/OrthologyOntology) despite the fact that no data are explicitly available about LOD on most of those DBs. FYI, probably we will release the ORTH 2 next month.

Further details are available in the figure below.
<img width="1416" alt="screen shot 2018-04-27 at 10 20 36" src="https://user-images.githubusercontent.com/15276341/39352321-aa5d23d8-4a04-11e8-8934-b57c88158e04.png">

**Comment by cmungall** [comment]
@thomaspd @huaiyumi ^^^

**Comment by cmungall** [comment]
Are there plans for unifying ORTH and HOM? It looks like ORTH is actually a homology ontology since I see parology in there :smiley:

ORTH seems more closely aligned to PANTHER at the moment.

Or perhaps two well delineated ontologies are intended? HOM is homology in the broader sense in the tradition of Owen, and can be used for anatomical structures, whereas I would guess ORTH is strict gene homology

**Comment by tarcisiotmf** [comment]
Yes you are right about the homology aspect. It was intended to have both ontologies. This is because we focus on different aspects (in my opinion). HOM ontology is more about a Taxonomy for Homology and ORTH ontology focus on being a "data schema" to represent the data from multiple orthology databases that also often include paralogy relations, sometimes as implicit information.   Those Orthology DBs are gene/protein centric, then as you noticed we focus on gene homology relations. 

Still we will align/link those two ontologies. However, in my opinion, we should not use equivalent axioms  between classes due to the fact that those terms are not exactly the same (e.g. orth:"homology relation"  not equal to hom:homology)
